src/Slices/ServiceInventory/UI/Components/RowActionsMenu/DestroyAction/DestroyAction.test.tsx

```diff
@@ -41,7 +41,19 @@ function setup() {
             commandResolver,
           }}
         >
-          <GetInstancesContext.Provider value={{ refetch }}>
+          <ServiceInventoryContext.Provider
+            value={{
+              labels: {
+                danger: [],
+                warning: [],
+                success: [],
+                info: [],
+                no_label: [],
+              },
+              onClick: jest.fn(),
```

Comment:
is the onClick not part of the labels logic?

Reply:
It's filtering callback, so it uses labels but per se I would keep it separated, I can change the name though to be more descriptive

Reply:
yes, because now it's not clear what the onClick does or with what it belongs. And for me it relates to the labels, it's when you click on a label/graph section.

Reply:
if it makes more sense you can also rename the group to : `labelFiltering`.

Reply:
changed